doc/build.txt

```diff
@@ -4,7 +4,7 @@ CONTENTS                                                         *build-contents
 
   1. Description..............................................|build-description|
   2. Commands....................................................|build-commands|
-    2.1 Build a specific target...........................................|Build|
+    2.1 Common tasks......................................................|Build|
     2.2 Initialize the current project................................|BuildInit|
     2.3 Print build informations for the current file.................|BuildInfo|
```

Comment:
Remove `:BuildInit` and `:BuildInfo` completely from this help page. The commands are deprecated now and only exist for backwards compatibility.